### PR TITLE
Implement APIv2 Exec Create and Inspect Endpoints

### PR DESCRIPTION
**Pull request by mheon**
Start and Resize require further implementation work.

WIP for now, until bindings and tests are added.

### CODE REVIEW

File: pkg/api/server/register_exec.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func (s *APIServer) registerExecHandlers(r *mux.Router) error {
-	// swagger:operation POST /containers/{name}/create compat createExec
+	// swagger:operation POST /containers/{name}/exec compat createExec
```
Comment (jwhonce) [inline pkg/api/server/register_exec.go]: If name may contain a '/' then

```suggestion
	// swagger:operation POST /containers/{name:.*}/exec compat createExec
```

Comment (mheon) [reply pkg/api/server/register_exec.go]: Container names allows letters, numbers, and underscores only

**Comment by mheon** [comment]
Added bindings. Tests are still red on my machine, trying to track down why.

**Comment by mheon** [comment]
Restarted flake, everything else green. @rhatdan @baude @vrothberg @TomSweeneyRedHat PTAL

File: pkg/bindings/test/exec_test.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		bt.RestoreImagesFromCache()
+		s = bt.startAPIService()
+		time.Sleep(1 * time.Second)
+		err := bt.NewConnection()
+		Expect(err).To(BeNil())
+	})
+
+	AfterEach(func() {
+		s.Kill()
+		//bt.cleanup()
```
Comment (vrothberg) [inline pkg/bindings/test/exec_test.go]: @mheon, do we need that?

Comment (mheon) [reply pkg/bindings/test/exec_test.go]: I have no idea. Copied verbatim from the other bindings tests.

Comment (vrothberg) [reply pkg/bindings/test/exec_test.go]: @baude, do we need the cleanup?

Comment (baude) [reply pkg/bindings/test/exec_test.go]: yup, it has been getting copied and pasted by me too ...

Comment (mheon) [reply pkg/bindings/test/exec_test.go]: Does this mean uncomment it? Or leave it commented?

Comment (baude) [reply pkg/bindings/test/exec_test.go]: uncomment it

Comment (mheon) [reply pkg/bindings/test/exec_test.go]: Fixed

**Pull request merged**